snaps/index.mdx

```diff
@@ -85,5 +85,5 @@ Features include:
 ## Questions?
 
 If you have questions about using MetaMask Snaps or want to propose a new feature, you can interact with the
-MetaMask Snaps team and community on [GitHub discussions](https://github.com/MetaMask/snaps-monorepo/discussions)
-and the Snaps Dev channel on [Consensys Discord](https://discord.gg/consensys).
+MetaMask Snaps team and community on [GitHub discussions](https://github.com/MetaMask/Snaps-monorepo/discussions)
```

Comment:
Oops, this URL is wrong

Reply:
It’s https://github.com/MetaMask/snaps/discussions

everything else is correct